goss-testing/scripts/check_k8s_loadbalancers.sh

```diff
@@ -53,23 +54,61 @@ function get_default_net_from_sls() {
       echo >&2 "error: failed to get Networks from SLS"
       exit 1
   fi
-
-  defaultRoute=$(echo "${NETWORKSJSON}" | jq -r --arg n "BICAN" '.[] | select(.Name == $n) | .ExtraProperties | .SystemDefaultRoute')
-  echo "$defaultRoute"
+  echo "$NETWORKSJSON"
 }
 
-default_bican_net=$(get_default_net_from_sls)
+function check_chn_config_from_sls() {
+
+  defaultRoute=$(echo "$1" | jq -r --arg n "BICAN" '.[] | select(.Name == $n) | .ExtraProperties | .SystemDefaultRoute')
```

Comment:
After the latest discussion on Friday and hearing that we will likely have cases where both CAN and CHN are defined in 1.3, I think it would be best to just handle that case now.   For each of `check_chn_config_from_sls`, `check_can_config_from_sls`, and `check_cmn_config_from_sls`, we should just simply check that the network is defined.
```
echo "$1" | jq -r --arg n "CMN" '.[] | select(.Name == $n)'
```

If the network is defined, then we should have pools and the LB services should have external IPs.